### ISSUE TITLE
[BUG] Using the config option --blank_line_after_tag should only affect lines that begin with the tag

### ISSUE THREAD

**Issue by christopherpickering** [Riverside-Healthcare/djLint#35]
- [x] I'm on the [latest version](https://pypi.org/project/djlint/) of djLint
- [x] I've searched the [issues](https://github.com/Riverside-Healthcare/djLint/issues)
- [x] I've read the [docs](https://djlint.com)

## Issue

For example, with a pyproject of:
```sh
[tool.djlint]
blank_line_after_tag="include"
```
his
```html
 <div class="tab-cnt">
     <div class="tab-dta active" id="details">
         <!-- this passes -->
         <div class="em-grid">{% include "pages/task/details_source.html.j2" %}
</div>
        <!-- but should be left as this -->
         <div class="em-grid">{% include "pages/task/details_source.html.j2" %}</div>
     </div>
 </div>
```
